verification/curator-service/ui/src/components/App.tsx

```diff
@@ -501,9 +508,13 @@ export default function App(): JSX.Element {
                                 <MenuIcon />
                             </IconButton>
                         )}
-                        <Link to="/" data-testid="home-button">
+                        <a
+                            href="https://dev-curator.ghdsi.org/"
```

Comment:
This button should redirect to homepage on marketing website (https://test-globalhealth.pantheonsite.io/). Not to the data homepage